src/main/kotlin/com/nikitanazarov/coroutinestacks/CoroutineStacksBundle.kt

```diff
@@ -14,6 +14,6 @@ object CoroutineStacksBundle : DynamicBundle(BUNDLE) {
     fun message(@NonNls @PropertyKey(resourceBundle = BUNDLE) key: String, vararg params: Any): String = getMessage(key, *params)
 }
 
-object CoroutineStackPanelData {
+object CoroutineStacksPanelData {
     var areLibraryFramesAllowed: Boolean = true
```

Comment:
That's a nice way to store contents like this when you have a large system where other components need to be aware of the state of each other. But since our plugin is small, let's move this constant to CoroutineStacksPanel